### ISSUE TITLE
Ability to create folders

### ISSUE THREAD

**Issue by OBellon** [christianalfoni/webpack-bin#148]
It would be nice as an enhancement.

**Comment by bebraw** [comment]
Yeah, I was just about to propose this. I think this would help a lot with the UI as you could nest files within directories and keep the root small. Quick mockup:

![screenshot 2016-06-09 18 07 13](https://cloud.githubusercontent.com/assets/166921/15934652/08c1c04c-2e6d-11e6-98d9-1109a22acfaf.png)

**Comment by OBellon** [comment]
That implementation would be awesome!

**Comment by christianalfoni** [comment]
Great idea guys :-)

**Comment by equinusocio** [comment]
Any update?!
